src/lib/constants.ts

```diff
@@ -41,8 +41,15 @@ export const COMMANDS = {
     SELECT_DEFAULT_ACCOUNT: 'hubspot.config.selectDefaultAccount',
     SET_DEFAULT_ACCOUNT: 'hubspot.config.setDefaultAccount',
   },
-  CREATE_MODULE: 'hubspot.create.module',
-  CREATE_SERVERLESS_FUNCTION_FOLDER: 'hubspot.create.serverlessFunctionFolder',
+  CREATE: {
+    MODULE: 'hubspot.create.module',
+    SECTION: 'hubspot.create.section',
+    TEMPLATE: 'hubspot.create.template',
+    PARTIAL: 'hubspot.create.partial',
+    GLOBAL_PARTIAL: 'hubspot.create.globalPartial',
+    SERVERLESS_FUNCTION: 'hubspot.create.serverlessFunction',
+    SERVERLESS_FUNCTION_FOLDER: 'hubspot.create.serverlessFunctionFolder',
+  },
```

Comment:
@miketalley cleaned this up a little, may require changes to tracking code (?)

Reply:
The tracking PR got merged in(https://github.com/HubSpot/hubspot-cms-vscode/pull/157). It looks like the tracked event for module creation would be removed in this PR. Can we add that one back? We should probably add tracked events for the other create commands as well.

Reply:
Added 👍 

I put them at the end of the callback and omitted the `await` since it shouldn't impact the rest of the logic and just needs to be fired off, see any issue with that?